crates/aptos-batch-encryption/Cargo.toml

```diff
@@ -0,0 +1,69 @@
+[package]
+name = "aptos-batch-encryption"
+version = "0.1.0"
+
+# Workspace inherited keys
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+publish = { workspace = true }
+repository = { workspace = true }
+rust-version = { workspace = true }
+
+[lib]
+name = "aptos_batch_encryption"
+doctest = false
+
+[dependencies]
+ark-bls12-381 = { workspace = true }
+ark-bn254 = { workspace = true }
+ark-ec = { workspace = true }
+ark-ff = { workspace = true }
+ark-ff-asm = { workspace = true }
+ark-ff-macros = { workspace = true }
+ark-poly = { workspace = true }
+ark-serialize = { workspace = true }
+ark-std = { workspace = true }
+ed25519-dalek = { version = "2.1.1", features = ["asm", "rand_core", "serde"] }
+sha2 = { version = "0.10.8", features = ["asm", "asm-aarch64", "sha2-asm"] }
+
```

Comment:
nit: I'd remove this space ,and have all dependencies sorted alphabetically. Not sure if makes senes to separate these into different groups (unless you have a reason?)